### DISCUSSION TITLE
Is headCollection needs when I'm use random commit id generator?

### DISCUSSION THREAD

**Discussion by gallix0116** [javers/javers#1297] [Q&A] [answered]
I'm considering using javers for my project with spring-boot-starter-data-mongodb.
My project is distributed application, so I'm going to use random commitid generator.
If I use random commitid generator, I don't think I need headCollection.

My concern is that headCollection seems to use one document, is there any problem in a distributed application environment? (e.g. write confilct)

At the issue, there is thinking the same thing as me -> https://github.com/javers/javers/issues/1208#issuecomment-1309057619

There seems to be no library like this for implementing Mongo Database Audit, so I'd like to use javers if possible, please reply

**Comment by bartoszwalacik** [comment, marked as answer]
You are right, in this case the headCollection seems obsolete

**Comment by bartoszwalacik** [comment]
fixed in https://github.com/javers/javers/issues/1208

**Reply by gallix0116** [reply]
@bartoszwalacik 
Hi.
Thank you for fixing it
fixed version 7.3.0 was based on Spring Boot 3 (java17).
By the way, our project is based on spring boot 2.x.x (java 11), so we can't use it... Can you release the javers 6.x.x version that reflects that change?

I'm not considering spring (boot) major version up(2->3) because of other side effects.

**Comment by bartoszwalacik** [comment]
Sorry @gallix0116 , no plans for developing 6.x. Maintaining 2 diverged main branches would be too time consuming for me
